cmd/maya-apiserver/spc-watcher/handler.go

```diff
@@ -84,9 +86,15 @@ func (c *Controller) spcEventHandler(operation string, spcGot *apis.StoragePoolC
 		// Hook Update Business Logic Here
 		return updateEvent, nil
```

Comment:
Why are we not returning error?

Reply:
Update is not handled now, this the case when RV of spc changes.(This will go in part of reconciling when RV changes of spc object.)
Hence it is so. If an err is returned object will be re-queued which we do not want to d it unnecessarily.